chapters/limits-and-specifications/supported-formats.mdx

```diff
@@ -81,5 +81,31 @@ Following these best practices will help you avoid issues due to limitations and
 | LinkedIn       | Video               | Released     |
 | Sharechat      | Video               | Released     |
 | Likee          | Video               | Released     |
-| TikTok (Beta)  | Video               | Beta          |
-| Twitter (Beta) | Video               | Beta          |
+| TikTok (Beta)  | Video               | Beta         |
+| Twitter (Beta) | Video               | Beta         |
```

Comment:
remove them ? We will never put them in "Released" state anyway.

Reply:
Pas bête, je me le note quand je repasse sur la doc!